docker/cheqd_node/mainnet_launch/docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: '3'
+
+services:
+  node:
+    image: cheqd-node
+    ports:
+      - "26656:26656" # p2p
+      - "26657:26657" # rpc
+    volumes:
+      - cheqd_data:/cheqd
+      - ../../../persistent_chains/mainnet/:/cheqd/.common
```

Comment:
Not sure of this path..., why does it need to look a few level up?

Reply:
I just don't want to copy/paste genesis and seeds into `mainnet_launch` directory. It's default behaviour and acan be changed by user of course.

Reply:
After today's discussion: this could be a variable in the .env file